site/styles/_variables.scss

```
@@ -32,11 +32,16 @@ $container-max-widths: (
 
 // Body / Core
 // ------------------------------------
+
+//To make dark or light theme change the two variables below
+$var-text: #000;
+$var-background: #fff;
```

Nit: `var` in the name is redundant.

```suggestion
$theme-color: #000;
$theme-background-color: #fff;
```

---

Would it not be cleaner to use the existing `$body-bg` and `$body-color` vars in all of the scss files?

---

I kinda agree, but two small things for the opposing view:

1. Reading `fill: $theme-color` is marginally improved on `fill: $body-color`
2. Can potentially group other vars under a `$theme-` prefix. This might be an entirely bogus reason if it turns out we don't need those news ones you suggested we don't need.

---

I reckon either use `$body-bg` and `$body-color` everywhere... or change all usage of those two to `$theme-something` in all the files.